release-notes/aws/v9.3.2.md

```diff
@@ -0,0 +1,9 @@
+# :zap: Giant Swarm Release v9.3.22 for AWS :zap:
+
+This release fixes an issue where upgrading from an earlier release would cause the `nginx-ingress-controller` service to have no endpoints.
```

Comment:
```suggestion
This release fixes an issue where upgrading from an earlier platform release to platform release v9.3.0 or v9.3.1 would cause the `nginx-ingress-controller` service to have no endpoints.
```